src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/__tests__/CreateAntigenTestProfileViewModelTests.swift

```diff
@@ -39,14 +50,28 @@ class CreateAntigenTestProfileViewModelTests: CWATestCase {
 			case .email:
 				viewModel.update("sabine.schulz@gmx.com", keyPath: \.email)
 				XCTAssertNotNil(viewModel.antigenTestProfile.email)
+			case .id:
+				break
 			}
 		}
 	}
 	
 	func testGIVEN_EmptyAntigenTestProfile_THEN_SetOneValueAndCheckIfEligibleToSave() {
 		// GIVEN
 		let store = MockTestStore()
-		let viewModel = AntigenTestProfileInputViewModel(store: store)
+		let antigenTestProfile = AntigenTestProfile(
+			firstName: "Max",
+			lastName: "Mustermann",
+			dateOfBirth: Date(timeIntervalSince1970: 390047238),
+			addressLine: "Blumenstraße 2",
+			zipCode: "43923",
+			city: "Berlin",
+			phoneNumber: "0165434563",
+			email: "sabine.schulz@gmx.com"
+		)
+		store.antigenTestProfiles = [antigenTestProfile]
```

Comment:
Same question here as above. Is the empty case tested somewhere else?

Reply:
Same.